app/src/main/res/values/strings.xml

```diff
@@ -10,4 +10,6 @@
     <string name="total_step_title">TOTAL STEP:</string>
     <string name="example_task_reward">20</string>
     <string name="example_task_description">Expose under the sun for 30 min and walk for 200km</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
That isn't needed anymore, right? If yes, remove it

Reply:
solved

Reply:
solved